### ISSUE TITLE
Minitest methods aren't being converted to RSpec `it`s

### ISSUE THREAD

**Issue by patientdev** [jaredbeck/minitest_to_rspec#6]
```bash
mt2rspec --version
=> 0.9.0
mt2rspec --rails test/controllers/webhooks_controller_test.rb
```

Input:

```ruby
require 'test_helper'

class WebhooksControllerTest < ActionController::TestCase
  def test_http_status_for_empty_request
    request = post :webhook_handler
    assert_equal request.status, 422
  end
end
```

Output:
```ruby
require("rails_helper")
RSpec.describe(WebhooksController, :type => :controller) do
  def test_http_status_for_empty_request
    request = post(:webhook_handler)
    expect(422).to(eq(request.status))
  end
end
```

**Comment by jaredbeck** [comment]
IIRC, we only support `test('foobar')` not `def test_foobar`. PRs welcome!

**Comment by patientdev** [comment]
Ah, cool. I'll look getting a PR together. Thanks.

**Comment by jaredbeck** [comment]
> Ah, cool. I'll look getting a PR together. Thanks.
> @patientdev patientdev closed this

Let's keep it open, if that's ok with you, to reduce the chance of a duplicate issue being created.

**Comment by jaredbeck** [comment]
Fixed by https://github.com/jaredbeck/minitest_to_rspec/pull/7, released in 0.10.0
